perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp

```diff
@@ -254,18 +252,46 @@ bool BicycleTracker::measureWithPose(
     R(0, 1) = 0.0;                  // x - y
     R(1, 1) = ekf_params_.r_cov_y;  // y - y
     R(1, 0) = R(0, 1);              // y - x
-    // R(2, 2) = ekf_params_.r_cov_yaw;                        // yaw - yaw
   } else {
     R(0, 0) = object.kinematics.pose_with_covariance.covariance[utils::MSG_COV_IDX::X_X];
     R(0, 1) = object.kinematics.pose_with_covariance.covariance[utils::MSG_COV_IDX::X_Y];
-    // R(0, 2) = object.kinematics.pose_with_covariance.covariance[utils::MSG_COV_IDX::X_YAW];
     R(1, 0) = object.kinematics.pose_with_covariance.covariance[utils::MSG_COV_IDX::Y_X];
     R(1, 1) = object.kinematics.pose_with_covariance.covariance[utils::MSG_COV_IDX::Y_Y];
-    // R(1, 2) = object.kinematics.pose_with_covariance.covariance[utils::MSG_COV_IDX::Y_YAW];
-    // R(2, 0) = object.kinematics.pose_with_covariance.covariance[utils::MSG_COV_IDX::YAW_X];
-    // R(2, 1) = object.kinematics.pose_with_covariance.covariance[utils::MSG_COV_IDX::YAW_Y];
-    // R(2, 2) = object.kinematics.pose_with_covariance.covariance[utils::MSG_COV_IDX::YAW_YAW];
   }
+
+  // if there are orientation available
+  if (dim_y == 3) {
+    // get compensated yaw measurement
+    double measurement_yaw = tier4_autoware_utils::normalizeRadian(
+      tf2::getYaw(object.kinematics.pose_with_covariance.pose.orientation));
+    {
+      Eigen::MatrixXd X_t(ekf_params_.dim_x, 1);
+      ekf_.getX(X_t);
+      while (M_PI_2 <= X_t(IDX::YAW) - measurement_yaw) {
+        measurement_yaw = measurement_yaw + M_PI;
+      }
+      while (M_PI_2 <= measurement_yaw - X_t(IDX::YAW)) {
+        measurement_yaw = measurement_yaw - M_PI;
+      }
+    }
```

Comment:
[IMO] this code is only used if orientation_availability is SIGN_UNKNOWN

Reply:
This is fixed in 590c0c692be75512805b9db689c81f3c036c920c .